src/Be.Vlaanderen.Basisregisters.DataDog.Tracing.AspNetCore/DataDogTracingApplicationBuilderExtensions.cs

```diff
@@ -51,6 +50,9 @@ public static class DataDogTracingApplicationBuilderExtensions
                     span.SetMeta("http.request.headers.host", context.Request.Host.ToUriComponent());
                     span.SetMeta("http.url", context.Request.GetEncodedUrl());
 
+                    if (options.LogForwardedForEnabled && context.Request.Headers.TryGetValue("X-Forwarded-For", out var values))
+                        span.SetMeta("http.request.headers.x-forwarded-for", values.ToString());
```

Comment:
Will ToString() give the correct values, or will it give a generic value for HttpRequestHeaders? I'm never sure.